packages/core/src/intl/languages.test.ts

```diff
@@ -133,51 +118,45 @@ describe('getLanguageFromStory', () => {
 
 describe('getCompanyInformation', () => {
     it('returns company information from target language', () => {
-        expect(getCompanyInformation(ALL_LANGUAGES, LocaleObject.fromAnyCode('nl-BE'))).toEqual(
+        expect(getCompanyInformation(ALL_LANGUAGES, 'nl-BE')).toEqual(
             LANGUAGES.nl_BE.company_information,
         );
     });
 
     it('returns company information from default language if target not found', () => {
-        expect(getCompanyInformation(ALL_LANGUAGES, LocaleObject.fromAnyCode('de-DE'))).toEqual(
+        expect(getCompanyInformation(ALL_LANGUAGES, 'de-DE')).toEqual(
             LANGUAGES.en.company_information,
         );
     });
 });
 
-describe('getShortestLocaleCode', () => {
+describe('getShortestLocaleSlug', () => {
     it('returns false for default language', () => {
-        expect(getShortestLocaleCode(ALL_LANGUAGES, LocaleObject.fromAnyCode('en'))).toBe(false);
+        expect(getShortestLocaleSlug(ALL_LANGUAGES, 'en')).toBe(false);
     });
 
     it('returns neutral language code if it is the only culture with that language', () => {
         expect(
-            getShortestLocaleCode(
+            getShortestLocaleSlug(
                 ALL_LANGUAGES.filter(({ code }) => code !== 'es_419'),
-                LocaleObject.fromAnyCode('es-ES'),
+                'es-ES',
             ),
         ).toBe('es');
     });
 
     it('returns region code if it is the only culture with that region', () => {
-        expect(getShortestLocaleCode(ALL_LANGUAGES, LocaleObject.fromAnyCode('en-US'))).toBe('US');
-        expect(getShortestLocaleCode(ALL_LANGUAGES, LocaleObject.fromAnyCode('en-GB'))).toBe('GB');
-        expect(getShortestLocaleCode(ALL_LANGUAGES, LocaleObject.fromAnyCode('nl-NL'))).toBe('NL');
-        expect(getShortestLocaleCode(ALL_LANGUAGES, LocaleObject.fromAnyCode('fr'))).toBe('FR');
+        expect(getShortestLocaleSlug(ALL_LANGUAGES, 'en-US')).toBe('us');
+        expect(getShortestLocaleSlug(ALL_LANGUAGES, 'en-GB')).toBe('gb');
+        expect(getShortestLocaleSlug(ALL_LANGUAGES, 'nl-NL')).toBe('nl-nl'); // cannot use `nl`, as it collides with language code
```

Comment:
UPD: I can see in the tests that it's not the case, so I assume we're only preventing collisions with enabled languages.

Reply:
> In this logic, the nl-BE will not shorten to be like it used to, right? (Colliding with be for Belarussian language code)

For `nl-BE` it depends:

- I can be shortened to `be`, if Belarusian is not enabled. 
- I won't be shortened to `be`, if Belarusian is enabled.

But for `nl-NL` it will never shorten to the region code. Because if `nl-NL` is enabled, the region code will always collide on the lang code.

Reply:
Okay, thanks for the explanation!